### PR TITLE
feat: Increase the client redirect_uri length limit

### PR DESCRIPTION
**Pull request by hsinn0**
- Change the database data type from varchar to text type as mysql varchar type is subject to the table's column size limit

[#183903458]

### CODE REVIEW

**Review by peterhaochen47 — APPROVED** [review]

File: uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/ClientAdminEndpointsIntegrationTests.java [hunk 1]
```diff
@@ -50,13 +50,7 @@
 import org.springframework.util.MultiValueMap;
 import org.springframework.web.client.RestTemplate;
 
-import java.util.ArrayList;
```
Comment (strehle) [inline uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/ClientAdminEndpointsIntegrationTests.java]: Can you please revert import changes.
There is an ongoning discussion about clean code but we have different view on that but therefore please exclude such changes where they are not needed

File: uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/clients/ClientAdminEndpointsMockMvcTests.java [hunk 2]
```diff
@@ -64,15 +64,7 @@
 import org.springframework.web.context.WebApplicationContext;
 
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
```
Comment (strehle) [inline uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/clients/ClientAdminEndpointsMockMvcTests.java]: See other comments about import changes

**Review by strehle — CHANGES_REQUESTED** [review]
please revert import changes

File: uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/ClientAdminEndpointsIntegrationTests.java (hunk 1 above)
Comment (hsinn0) [reply uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/ClientAdminEndpointsIntegrationTests.java]: Done

File: uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/clients/ClientAdminEndpointsMockMvcTests.java (hunk 2 above)
Comment (hsinn0) [reply uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/clients/ClientAdminEndpointsMockMvcTests.java]: Done

**Comment by peterhaochen47** [comment]
Merged as all comments have been addressed.

**Pull request merged**